fluentforms/core/src/main/java/com/_4point/aem/fluentforms/testing/assembler/MockTraditionalAssemblerService.java

```diff
@@ -17,17 +16,24 @@
 import com.adobe.fd.assembler.client.PDFAValidationResult;
 
 public class MockTraditionalAssemblerService implements TraditionalDocAssemblerService {
-	
+
 	private final AssemblerResult DUMMY_ASSEMBLER_RESULT;
```

Comment:
Since this is a constant, it is more typically initialized in the variable declaration.  The approach you've chosen works, but it is unusual.

Reply:
not sure what exactly needs to be done here.. MockTraditionalOuput's DUMMY_BATCH_RESULT  is created in the same way..I took reference from there